src/cctag/Params.hpp

```diff
@@ -180,6 +184,10 @@ struct Parameters
     uint32_t _maxEdges;
     ///  if compiled with CCTAG_WITH_CUDA, whether to use cuda algorithm or not, otherwise it is ignored
     bool _useCuda;
+    /// if _useCuda, physical memory reserved for internal counters, otherwise unused
```

Comment:
maybe to be more explicit add what the number represents, MB, kB or else

Reply:
comment added